control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1451,6 +1413,17 @@ func (r *reconciler) reconcileCloudCredentialSecrets(ctx context.Context, hcp *h
 	return errs
 }
 
+func AzureFederatedTokenCredentialSecretData(name, infraID string, config *hyperv1.AzurePlatformSpec) map[string][]byte {
+	return map[string][]byte{
+		"azure_region":               []byte(config.Location),
+		"azure_resource_prefix":      []byte(name + "-" + infraID),
```

Comment:
The conversation [here](https://github.com/openshift/cloud-credential-operator/blob/master/docs/azure_workload_identity.md) does not explicitly list the resource prefix and resource-group, but I can't imagine it will hurt?